src/_tkagg.cpp

```diff
@@ -100,39 +88,33 @@ static int PyAggImagePhoto(ClientData clientdata, Tcl_Interp *interp, int
     }
 
     /* check for bbox/blitting */
-    if (sscanf(argv[4], SIZE_T_FORMAT, &bboxl) != 1) {
-        TCL_APPEND_RESULT(interp, "error casting pointer", (char *)NULL);
-        return TCL_ERROR;
-    }
-    bboxo = (PyObject *)bboxl;
-
-    if (bboxo != NULL && bboxo != Py_None) {
-        agg::rect_d rect;
-        if (!convert_rect(bboxo, &rect)) {
-            return TCL_ERROR;
-        }
-
+    if (sscanf(argv[4], BBOX_FORMAT, &x1, &x2, &y1, &y2) == 4) {
         has_bbox = true;
+    }
+    else {
+        has_bbox = false;
+    }
 
-        destx = (int)rect.x1;
-        desty = srcheight - (int)rect.y2;
-        destwidth = (int)(rect.x2 - rect.x1);
-        destheight = (int)(rect.y2 - rect.y1);
+    if (has_bbox) {
+        int srcstride = wdata * 4;
+        destx = x1;
+        desty = hdata - y2;
+        destwidth = x2 - x1;
+        destheight = y2 - y1;
         deststride = 4 * destwidth;
 
-        destbuffer = new agg::int8u[deststride * destheight];
+        destbuffer = new uint8_t[deststride * destheight];
```

Comment:
any reason for the change?

Reply:
now it no longer uses ``#include py_converters.h`` which supplied the ``agg:int8u``